quadkey/bq/sql/ST_ASQUADINTPOLYFILL.sql

```diff
@@ -0,0 +1,28 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_QUADKEY@@.POLYFILL_FROMGEOJSON`
+    (geojson STRING, resolution INT64)
+    RETURNS ARRAY<INT64>
+    DETERMINISTIC
+    LANGUAGE js
+    OPTIONS (library=["@@QUADKEY_BQ_LIBRARY@@"])
+AS """
+    if(!geojson || resolution == null)
+    {
+        throw new Error('NULL argument passed to UDF');
+    }
+
+    let pol = JSON.parse(geojson);
+    let quadints = geojsonToQuadints(pol, {min_zoom: resolution, max_zoom: resolution});
+    return quadints.map(String);
+""";
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_QUADKEY@@.ST_ASQUADINTPOLYFILL`
```

Comment:
```suggestion
CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_QUADKEY@@.ST_ASQUADINT_POLYFILL`
```

For consistency with `ST_ASH3_POLYFILL`